app/db/models/document/physical_document.py

```diff
@@ -0,0 +1,45 @@
+import sqlalchemy as sa
+from app.db.session import Base
+
+
+class PhysicalDocument(Base):
+    """A physical document.
+
+    Representation of a document that exists in the real world.
+    Owned and updated only by the pipeline.
+    """
+
+    __tablename__ = "physical_document"
+
+    id = sa.Column(sa.Text, primary_key=True)
+    title = sa.Column(sa.Text, nullable=False)
+    md5_sum = sa.Column(sa.Text, nullable=True)
+    source_url = sa.Column(sa.Text, nullable=True)
```

Comment:
I know I set this to `varchar` in the table, but maybe we can use a URL type here?

Reply:
ok, this seems to require a custom extension, so scratch that idea